src/App.js

```diff
@@ -1,5 +1,24 @@
-function App() {
-  return <div className="App"></div>;
+import { useState } from 'react';
+import styled from 'styled-components';
+import Wishlist from './Wishlist.js';
+import AddDiveWish from './AddDiveWish.js';
+
+export default function App() {
+  const [diveWishes, setDiveWishes] = useState([]);
+
+  return (
+    <>
+      <Heading>Diving Wishlist</Heading>
+      <AddDiveWish onAddDiveWish={AddToWishlist} />
+      <Wishlist diveWishes={diveWishes} />
+    </>
+  );
+
+  function AddToWishlist(destination, notes) {
+    setDiveWishes([{ destination: destination, notes: notes }, ...diveWishes]);
+  }
```

Comment:
As I suggested earlier, I think I would already get an object here and not create it in your `addToWishList` function. But again, I think that's just my personal preference :)